plugins/inputs/influxdb_v2_listener/influxdb_v2_listener_test.go

```diff
@@ -583,6 +584,22 @@ func TestHealth(t *testing.T) {
 	require.EqualValues(t, 4, listener.healthsServed.Get())
 }
 
+func TestPing(t *testing.T) {
+	listener := newTestListener()
+	acc := &testutil.Accumulator{}
+	require.NoError(t, listener.Init())
+	require.NoError(t, listener.Start(acc))
+	defer listener.Stop()
+
+	resp, err := http.Get(createURL(listener, "http", "/ping", ""))
+	require.NoError(t, err)
+	require.EqualValues(t, 204, resp.StatusCode)
+	require.NoError(t, resp.Body.Close())
+	require.EqualValues(t, 1, listener.pingServed.Get())
+	require.EqualValues(t, internal.FormatFullVersion(), resp.Header.Get("X-Influxdb-Version"))
+	require.EqualValues(t, "Telegraf", resp.Header.Get("X-Influxdb-Build"))
```

Comment:
Keep these with the other response checks. Maybe also check if the Body is empty?